### PR TITLE
Differentiate response errors

### PR DESCRIPTION
**Pull request by niro1987**
Please review these changes.

request wrapper now differentiates between different types of errors.
- ResponseError in case of 5** resposne
- LoginError in case of 4** response
- RequestError in case of connection issues
- Error("Unknown error") in case of any response <> status_code 200

### CODE REVIEW

File: verisure/session.py [hunk 1]
```diff
@@ -100,16 +101,20 @@ def wrapper(url, *args, **kwargs):
...
-                            self._base_urls.reverse()
-                            continue
                         return response
                     response.raise_for_status()
                 except requests.exceptions.RequestException as ex:
-                    last_exception = ex
+                    last_exception = RequestError(str(ex))
                 self._base_urls.reverse()
-            raise Error from last_exception
+            raise last_exception
```
Comment (persandstrom) [inline verisure/session.py:116]: This change will let errors from the requests library slip through. Maybe you can wrap raise_for_status?

Comment (niro1987) [reply verisure/session.py:116]: rais_for_status could be removed. Looking at the definition, it checks the same status codes. Would you agree?

Comment (persandstrom) [reply verisure/session.py:116]: True, it can be removed. The check is from 400 to 600.

**Comment by persandstrom** [comment]
Something is missing now. A default action must be taken if non of the statements  are true. An exception for unknown response?

**Comment by niro1987** [comment]
The default action is the Unknown Error

**Comment by persandstrom** [comment]
You are correct!

**Pull request merged**